docs/iris/src/whatsnew/latest.rst

```diff
@@ -27,7 +27,9 @@ Features
   dependency on `matplotlib`_ at ``v2.x``.  Now that ``Python2`` support has
   been dropped, ``Iris`` is free to use the latest version of `matplotlib`_.
 
-* `CF Ancillary Data`_ variables are now supported.
+* `CF Ancillary Data`_ variables are now supported, and can be loaded from and
```

Comment:
would it be useful to link to an issue or pr?

Reply:
@tkknight Glad to do that, but for consistency that should also be the case for all other `whatsnew` entries...

Reply:
I think it would make the whats new page more useful - we could keep this discussion separate to this PR so we do not block it.

Reply:
@tkknight Cool. 

For this PR I'll add the relevant links per `whatsnew` entry mentioned here. Then I'll raise a separate PR to add relevant links to all the other entries, if possible... just to separate concerns. Thanks 👍